backend/src/v5/models/modelSettings.js

```diff
@@ -34,6 +38,38 @@ const getModelByQuery = async (ts, query, projection) => {
 	return res;
 };
 
+Models.addModel = async (ts, data) => {
+	const res = await insertOneModel(ts, { _id: generateUUIDString(), ...data });
+	return res.insertedId;
```

Comment:
I think it's safer to just assign the id to a variable and return that, instead of relying on mongo giving you back something you passed in?